dicom_utils/cli/dicom2img.py

```diff
@@ -80,19 +118,25 @@ def dicom_to_image(
             frames[0].save(path, save_all=True, append_images=frames[1:], duration=duration_ms, quality=quality)
 
 
+def path_to_sources(path: Path) -> List[Path]:
+    if path.is_dir():
+        return [f for f in path.iterdir() if has_dicm_prefix(f)]
+    if path.is_file():
+        return [path]
+    else:
+        raise FileNotFoundError(path)
+
+
 def main(args: argparse.Namespace) -> None:
-    source = Path(args.file)
+    path = Path(args.path)
     dest = Path(args.output) if args.output is not None else None
 
-    if not source.is_file():
-        raise FileNotFoundError(source)
-
     # handle case where output path is a dir
     if dest is not None and dest.is_dir():
-        dest = Path(dest, source.stem).with_suffix(".png")
+        dest = Path(dest, path.stem).with_suffix(".png")
 
-    with pydicom.dcmread(source) as dcm:
-        dicom_to_image(dcm, dest, args.split, args.fps, args.quality, not args.noblock)
+    dcms = [pydicom.dcmread(source) for source in path_to_sources(path)]
```

Comment:
Should a try/except block be added to `dcms` list building? If we want to visualize a directory but that directory contains one badly-formatted / unreadable dicom, we won't be able to visualize anything. Seems preferable to log such errors and continue on, visualizing whatever we were able to read.

Reply:
Yes, I think that makes sense. I'll add that in.